NAMESPACE

```diff
@@ -33,6 +34,8 @@ export(metadata_update_taxonomic_change)
 export(plot_trait_distribution_beeswarm)
 export(read_csv_char)
 export(read_metadata)
+export(trait_pivot_longer)
```

Comment:
@dfalster What should be the naming convention for these functions? `trait_table_pivot_wider`?

Reply:
Comes back to the discussion we were have the other day. What's the prefix for any function using the results of a {traits.build} pipeline. How about `db_traits_pivot_longer`?